docs/_content/ontwikkelaars/index.md

```diff
@@ -1,25 +1,36 @@
 ---
 title: Ontwikkelaars
 ---
-De ZGW API-stanadaarden worden gedefineerd in [de OAS3-specificatie en het voorgeschreven runtime-gedrag](../standaard/index). Om ontwikkelaars bij gemeentes en leveranciers te helpen met het realiseren
+<!-- De ZGW API-standaarden worden gedefineerd in [de OAS3-specificatie en het voorgeschreven runtime-gedrag](../standaard/index). Om ontwikkelaars bij gemeentes en leveranciers te helpen met het realiseren  -->
+De ZGW API-standaarden worden gedefineerd in [de OAS3-specificatie en het voorgeschreven runtime-gedrag](../standaard/index.md). Om ontwikkelaars bij gemeentes en leveranciers te helpen met het realiseren
 van, en aansluiten op, verschillende producten die volgens de ZGW API-standaarden
 werken, zijn naast de bovengenoemde stanadaardspecificaties, voor de verschillende API's referentie-implementaties gerealiseerd. Hierbij zijn diverse handeidingen en tutorials gemaakt. Hieronder een overzicht:
 
 ## Handleidingen
 
 ### Algemene handleidingen
-De handleidingen vormen een algemene introductie voor ontwikkelaars die met de ZGW API's aan de slag willen.
+<!-- De handleidingen vormen een algemene introductie voor ontwikkelaars die met de ZGW API's aan de slag willen.
```

Comment:
Beter oude versie weghalen ipv commenten, toch?

Reply:
Idee achter commenten was om alleen de oude links te commenten en daar de nieuwe links voor in plaats te zetten. Daardoor kunnen we straks eenvoudig weer terugschakelen naar de huidige situatie.